Framework/CurveFitting/src/IMWDomainCreator.cpp

```diff
@@ -145,45 +145,42 @@ std::pair<size_t, size_t> IMWDomainCreator::getXInterval() const {
 
   setParameters();
 
-  // find the fitting interval: from -> to
+  // From points to the first occurrence of StartX in the workspace interval.
+  // End points to the last occurrence of EndX in the workspace interval.
```

Comment:
@mantid-roman Would you like that commenting updating but apart from that this is okay?